docs/diagrams/AddEventSequenceDiagram.puml

```diff
@@ -0,0 +1,78 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":AddEventCommandParser" as AddEventCommandParser LOGIC_COLOR
+participant ":AddEventCommand" as AddEventCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("addevent 1 ed/Tutorial da/2022-03-14")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("addevent 1 ed/Tutorial da/2022-03-14")
+activate AddressBookParser
+
+create AddEventCommandParser
+AddressBookParser -> AddEventCommandParser
+activate AddEventCommandParser
+
+AddEventCommandParser --> AddressBookParser
+deactivate AddEventCommandParser
+
+AddressBookParser -> AddEventCommandParser : parse("1 ed/Tutorial da/2022-03-14")
+activate AddEventCommandParser
+
+create AddEventCommand
+AddEventCommandParser -> AddEventCommand : AddEventCommand(index, eventToAdd)
+activate AddEventCommand
+
+AddEventCommand --> AddEventCommandParser
+deactivate AddEventCommand
+
+AddEventCommandParser --> AddressBookParser
+deactivate AddEventCommandParser
+
+AddressBookParser --> LogicManager
+deactivate AddressBookParser
+
+LogicManager -> AddEventCommand : execute(model)
+activate AddEventCommand
+
+AddEventCommand -> AddEventCommand : createEditedSchedule(scheduleToEdit, eventToAdd)
+activate AddEventCommand
+
+AddEventCommand --> AddEventCommand : updatedSchedule
+deactivate AddEventCommand
+
+AddEventCommand -> AddEventCommand : createEditedPerson(personToEdit, editPersonDescriptor)
```

Comment:
This one I'm not sure, but one thing I want to ask is that is it OK if the scheduleToEdit and personToEdit weren't specified where they came from?